src/tls/TLSClient.fst

```diff
@@ -0,0 +1,298 @@
+module TLSClient
```

Comment:
Can you add a one-liner saying what's the difference between `TLSClient.fst` and `TLSClient13.fst`?